app/lib/scraper.rb

```diff
@@ -0,0 +1,62 @@
+class Scraper
+
+
+
+  def self.fetch
+
+
+    journal_feeds = JournalFeed.all
+
+    feed = Feedjira::Feed.fetch_and_parse(journal_feeds.last.url)
```

Comment:
What is this `feed` local being used for?

Reply:
The feed represents the information being pulled from each journal_feed's rss site. I changed the name of this variable to rss_feed for greater clarity in a later branch.